set.mm

```diff
@@ -58788,21 +58833,49 @@ singleton of the first member (with no sethood assumptions on ` B ` ).
     ( wcel ccnv csn cima cin cpred elinel1 df-pred eleq2s ) DAEDABFCGHZIABCJDAN
     KABCLM $.
 
+  ${
+    $d x A $.  $d x R $.  $d x X $.  $d x Y $.
+    $( Closed form of ~ elpredim .  (Contributed by Scott Fenton, 13-Apr-2011.)
+       (Revised by NM, 5-Apr-2016.) $)
+    predbrg $p |- ( ( X e. V /\ Y e. Pred ( R , A , X ) ) -> Y R X ) $=
+      ( vx wcel cpred wbr cv wi wceq predeq3 eleq2d imbi12d vex elpredim vtoclg
+      breq2 imp ) DCGEABDHZGZEDBIZEABFJZHZGZEUDBIZKUBUCKFDCUDDLZUFUBUGUCUHUEUAE
+      ABUDDMNUDDEBSOABUDEFPQRT $.
+  $}
+
+  ${
+    $d A y z $.  $d R y z $.  $d X y z $.  $d Y y z $.
+    $( If ` R ` is transitive over ` A ` and ` Y R X ` , then
+       ` Pred ( R , A , Y ) ` is a subclass of ` Pred ( R , A , X ) ` .
+       (Contributed by Scott Fenton, 28-Oct-2024.) $)
+    predtrss $p |- (
+     ( ( ( R i^i ( A X. A ) ) o. ( R i^i ( A X. A ) ) ) C_ R /\
+       Y e. Pred ( R , A , X ) /\ X e. A ) ->
```

Comment:
Wouldn't it make it more usable to write that conjunct as `Y R X` instead of `Y e. Pred ( R , A , X )` ? (same below)

Reply:
You got to it before I did 😄